salt_sproxy/cli.py

```diff
@@ -195,15 +204,17 @@ def run(self):
             'no_cached_pillar': 'use_cached_pillar',
             'no_grains': 'with_grains',
             'no_pillar': 'with_pillar',
-            'no_target_cache': 'target_cache'
+            'no_target_cache': 'target_cache',
         }
         for opt, kwarg in six.iteritems(reverse_opts):
             if getattr(self.options, opt):
                 kwargs[kwarg] = False
         kwargs['events'] = self.config.get('events', False)
         kwargs['use_existing_proxy'] = self.config.get('use_existing_proxy', False)
         kwargs['test_ping'] = self.config.get('test_ping', False)
-        kwargs['target_cache_timeout'] = self.config.get('target_cache_timeout', 60)  # seconds
+        kwargs['target_cache_timeout'] = self.config.get(
+            'target_cache_timeout', 60
```

Comment:
Aren't these defaults should be beforehand prepared in `self.config`?

Reply:
Nah, in `self.config` it's only what's in the config file + CLI options.